app/helpers/view_helper.rb

```diff
@@ -151,6 +151,10 @@ def base_errors_hash(provider_code, course)
         provider_recruitment_cycle_references_path(provider_code, course.recruitment_cycle_year),
       "You must provide a UK provider reference number (UKPRN) and URN" =>
         provider_recruitment_cycle_references_path(provider_code, course.recruitment_cycle_year),
+      "You must provide a UK Provider Reference Number (UKPRN)" =>
+        provider_recruitment_cycle_references_path(provider_code, course.recruitment_cycle_year),
+      "You must provide a UK Provider Reference Number (UKPRN) and URN" =>
+        provider_recruitment_cycle_references_path(provider_code, course.recruitment_cycle_year),
```

Comment:
Trying to move away from using ‘You must’ in validation strings:

```suggestion
      "Enter a UK Provider Reference Number (UKPRN)" =>
        provider_recruitment_cycle_references_path(provider_code, course.recruitment_cycle_year),
      "Enter a UK Provider Reference Number (UKPRN) and URN" =>
        provider_recruitment_cycle_references_path(provider_code, course.recruitment_cycle_year),
```

Reply:
🤦 We should really settle on what these messages are supposed to be or we're going to be doing this forever and it's a faff. These strings have to match TTAPI for the functionality to work (I know 🤮 but this is the world we currently live in). @gregknight1 Could you just confirm and I'll sort it out?

Reply:
@gpeng @paulrobertlloyd Yep, 'Enter..' is the right instruction. Just fyi - I have tickets to review validation messages in Publish and Register. And also a ticket to review capitalisation in both services - they just keep getting pushed back due to transition stuff.

Reply:
I've spoken to @gregknight1 . We're going to update to `Enter` as you suggest @paulrobertlloyd (and in one of the preceding messages that isn't in this PR yet) and then @gregknight1 will review capitalisation etc as a wider bit of work to get it all consistently correct.

Reply:
Updated in e192ee2